modules/programs/tmux/default.nix

```diff
@@ -99,7 +99,9 @@ in
 
   config = mkIf cfg.enable (mkMerge [
     # add all plugins installed by themes
-    { soxin.programs.tmux.plugins = cfg.theme.plugins; }
+    (optionalAttrs (mode == "NixOS" || mode == "home-manager") {
+      soxin.programs.tmux.plugins = cfg.theme.plugins;
```

Comment:
Nothing uses `tmux.plugins` in Darwin if I'm not mistaken. Why was this needed?

Reply:
Not darwin but home-manager on darwin. My tmux works fine on Darwin right now, themes and all. Without this change, this was being set for nix-darwin that does not have the option.